### DISCUSSION TITLE
Disabling Scratch Layer?

### DISCUSSION THREAD

**Discussion by RobLW** [paperwm/PaperWM#846] [Q&A] [answered]
Hi there,

Is it possible to disable the scratch layer please?

I'm new to tiling, really enjoying PaperWM but I keep having old habit of minimising applications, or still just grabbing my mouse to select an app on my task bar and accidentally minimise.
When I bring them back up they are in the scratch layer and I always faff about trying to to put the window back into the tiling space. I forget the key stroke, then I miss the fact that my selected window isn't the one on the scratch layer but the one behind it, so I end up putting that into the scratch layer..... My ADHD brain fails me, and my muscle memory isn't yet trained.

I appreciate PaperWM has been designed to put apps into a scratch layer when it's been pulled out of the tiling (inc minimised). I feel being able to disable the scratch layer all together with a flip switch in settings would allow me to focus on a smaller set of skills to learn first, then re-introduce it when/if I'm ready for a scratch layer.
Hopefully I'm not the only one who might find such an option to be useful. 

Many thanks

**Comment by Lythenas** [comment, marked as answer]
I believe your problem should already be fixed by #840 

What gnome version are you on? Can you try updating the PaperWM version?

**Comment by RobLW** [comment]
Amazing! thank you.
I just updated from PaperWM 46.6.4 to 46.6.7 just now. Works a treat! :)
